### PR TITLE
Day12

### PR DESCRIPTION
**Pull request by ap221882**
Adding Login Route and enabling login user info on navbar - USING REDUX TOOLKIT

### CODE REVIEW

File: Day12/github-api-project-ts/src/pages/AuthPages/Login/LoginInputs.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { Input } from "../../../components";
+import { StyledFileInput } from "./login.styles";
+import { IInputProps, IFileInputProps } from "./login.types";
+
```
Comment (Pujarini) [inline Day12/github-api-project-ts/src/pages/AuthPages/Login/LoginInputs.tsx:4]: great way of putting Input types in different files

File: Day12/github-api-project-ts/src/routes/index.tsx [hunk 2]
```diff
@@ -42,9 +42,11 @@ export const myAppRouter = createBrowserRouter([
   {
     path: "/login",
     element: <Login />,
```
Comment (Pujarini) [inline Day12/github-api-project-ts/src/routes/index.tsx:44]: error element you can add when you are referencing some backend data for existing user else i don't think this case will arise.

File: Day12/github-api-project-ts/src/routes/index.tsx [hunk 3]
```diff
@@ -42,9 +42,11 @@ export const myAppRouter = createBrowserRouter([
...
+    errorElement: <ErrorPage />,
   },
   {
     path: "/logout",
     element: <Logout />,
+    errorElement: <ErrorPage />,
```
Comment (Pujarini) [inline Day12/github-api-project-ts/src/routes/index.tsx:50]: same for here also.

**Review by Pujarini — CHANGES_REQUESTED** [review]
reviewed!

File: Day12/github-api-project-ts/src/routes/index.tsx (hunk 2 above)
Comment (ap221882) [reply Day12/github-api-project-ts/src/routes/index.tsx:44]: Actually, I got an error while loading the Login Page and the same react-router error that I should have an ErrorComponent to handle, so I've added it. Yes, it was an error from Frontend Only.

**Comment by ap221882** [comment]
> reviewed!

Thanks @Pujarini!

**Review by nam334 — COMMENTED** [review]
reviewed. nice work!

**Pull request merged**